lib/markdown2.py

```diff
@@ -347,6 +348,9 @@ def convert(self, text):
 
         text = self.preprocess(text)
 
+        if "wavedrom" in self.extras:
```

Comment:
I see mixed quotes and ticks throughout the patch, can you opt for one? I know the rest of the file is probably mixed too though

Reply:
I've updated to use single quotes as consistently as possible.
The rest of the file is pretty mixed, I agree. Perhaps at some point a style should be picked and enforced with something like `flake8`